release/app/package.json

```diff
@@ -1,18 +1,18 @@
 {
   "name": "route-ui",
-  "version": "1.0.0",
```

Comment:
dont do this as part of the PR. version bump shoudl be a separate commit and there should be nothing happening after that for that version